### PR TITLE
fix _macro_init definitions (hotfix)

### PR DESCRIPTION
**Pull request by Alex-Jordan**
These were not quite right in the move from the OPL.

### CODE REVIEW

**Review by drgrice1 — APPROVED** [review]
Good find on this.  I should have caught this earlier.

**Review by drgrice1 — APPROVED** [review]

File: macros/math/algebraMacros.pl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # define any custom subroutines you want and then use them in
 # problems by including the file in loadMacros() calls.
 
-sub _init_macro_template {
```
Comment (pstaabp) [inline macros/math/algebraMacros.pl:6]: Doesn't this need to have the name of the macro? So `_algebra_macros_init` ?

**Comment by drgrice1** [comment]
Actually, it needs to be `_algebraMacros_init`.

**Comment by Alex-Jordan** [comment]
Changed on both PRs and force pushed.

**Pull request merged**